### DISCUSSION TITLE
Where Release?

### DISCUSSION THREAD

**Discussion by semi710** [PerformanC/ReZygisk#50] [Q&A] [answered]
.

**Comment by ThePedroo** [comment, marked as answer]
A release will be made once the major changes come, that is zygiskd c99 and web UI, together with stability.

**Comment by Keeperorowner** [comment]
> A release will be made once the major changes come, that is zygiskd c99 and web UI, together with stability.

I noticed that the release version has been released. Congratulations

**Reply by ThePedroo** [reply]
Changes in the plan, release won't be with the Web UI to not delay it any further.

**Reply by Keeperorowner** [reply]
> Changes in the plan, release won't be with the Web UI to not delay it any further.

Why not just use the zygisk_mod webui, which is already mature

**Reply by ThePedroo** [reply]
If you compare the ReZygisk Web UI with zygisk_mod Web UI, it's easy to see why: RZ one is ahead of zygisk_mod's when it comes to following APatch/KernelSU theme and being more complete.

We also focus on following Material Design 3, which zm doesn't.

![IMG_20241111_114345_175.jpg](https://github.com/user-attachments/assets/378e594c-c4e5-4648-8c73-f3352193e2b9)

![IMG_20241111_114412_247.jpg](https://github.com/user-attachments/assets/38961f72-5ee1-4dec-b01b-b16f58e4ce7d)

OBS: Zygisk_mod Web UI *may* be outdated in that photo, however it's the only one I found.

**Reply by Qing0731** [reply]
The last version of the zygisk_mod WebUI works like this

![Screenshot_2024-11-12-01-22-02-40_26ccd656f4ada08d09da6514b90ed2c0.jpg](https://github.com/user-attachments/assets/425daa82-2aa6-4efa-8adc-1c5b850d12cb)

And Zygisk_mod has not been maintained for a long time

**Reply by ThePedroo** [reply]
In any way, IMO I prefer RZ's one over zygisk_mod's and ZN's Web UI (not really biased, it's just that it is more organized and actually follows the design apps are tending to follow).
